VictorMaria/Banka
# #165262993 Build out getUser endpoint

#### What does this PR do?

Has getUser endpoint

#### Description of Task to be completed?

Has GET /api/v1/users/{id} working

#### How should this be manually tested?

1. Clone the repo and cd into the directory

2. Run npm start and launch Postman

3. Paste http://localhost:3000/api/v1/user/1 in the URL bar

4. Select GET method and click send

5. Wait for response which should be a json item containing the user's details who's id is 1

#### What are the relevant pivotal tracker stories?

#165262993
